airbyte-webapp/src/components/SimpleTableComponents/SimpleTableComponents.tsx

```diff
@@ -5,7 +5,7 @@ export const Row = styled.div`
   flex-direction: row;
   justify-content: flex-start;
   align-items: center;
-  height: 32px;
+  min-height: 70px;
```

Comment:
@YatsukBogdan1 I think this change has messed up some of the styling on the Notifications settings page. E.g. when I run master locally, I see this:
<img width="1222" alt="image" src="https://user-images.githubusercontent.com/22731524/192371831-75a618b7-c3ef-4772-837a-3f3516182dc8.png">
the spacing here is large because the `min-height` is being applied to the fields in that screenshot, which I believe is not desired